src/Bridge/Symfony/Bundle/Command/DebugResourceCommand.php

```diff
@@ -0,0 +1,114 @@
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure(): void
+    {
+        $this
+            ->setDescription('Debug API Platform resources')
+            ->addArgument('class', InputArgument::REQUIRED, 'The class you want to debug');
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $resourceClass = $input->getArgument('class');
+
+        $resourceCollection = $this->resourceMetadataCollectionFactory->create($resourceClass);
+
+        if (0 === \count($resourceCollection)) {
+            $output->writeln(sprintf('<error>No resources found for class %s</error>', $resourceClass));
+
+            return 1;
+        }
+
+        $shortName = (false !== $pos = strrpos($resourceClass, '\\')) ? substr($resourceClass, $pos + 1) : $resourceClass;
+
+        $helper = $this->getHelper('question');
+
+        $resources = [];
+        foreach ($resourceCollection as $resource) {
+            if ($resource->getUriTemplate()) {
+                $resources[] = $resource->getUriTemplate();
+                continue;
+            }
+
+            foreach ($resource->getOperations() as $operation) {
+                if ($operation->getUriTemplate()) {
+                    $resources[] = $operation->getUriTemplate();
+                    break;
+                }
+            }
+        }
+
+        if (\count($resourceCollection) > 1) {
+            $questionResource = new ChoiceQuestion(
+                sprintf('There are %d resources declared on the class %s, which one do you want to debug ? ', \count($resourceCollection), $shortName).\PHP_EOL,
+                $resources
+            );
+
+            $answerResource = $helper->ask($input, $output, $questionResource);
+            $resourceIndex = array_search($answerResource, $resources, true);
+            $selectedResource = $resourceCollection[$resourceIndex];
+        } else {
+            $selectedResource = $resourceCollection[0];
+            $output->writeln(sprintf('Class %s declares 1 resource.', $shortName).\PHP_EOL);
+        }
+
+        $operations = ['Debug the resource itself'];
+        foreach ($selectedResource->getOperations() as $operationName => $operation) {
+            $operations[] = $operationName;
+        }
+
+        $questionOperation = new ChoiceQuestion(
+            sprintf('There are %d operation%s declared on the resource, which one do you want to debug ? ', $selectedResource->getOperations()->count(), $selectedResource->getOperations()->count() > 1 ? 's' : '').\PHP_EOL,
+            $operations
+        );
+
+        $answerOperation = $helper->ask($input, $output, $questionOperation);
+        if ('Debug the resource itself' === $answerOperation) {
+            dump($selectedResource);
+
+            return Command::SUCCESS;
+        }
+
+        dump($resourceCollection->getOperation($answerOperation));
```

Comment:
This function is usually not available in prod. Maybe should we use the Var Dumper component directly instead?

Reply:
We declared this in the `debug.xml` file which is loaded only in debug mode but I agree. It'll be also easier to test (and tests are missing).